src/components/connectAdvanced.js

```diff
@@ -322,6 +323,8 @@ export default function connectAdvanced(
 
           // If the child props haven't changed, nothing to do here - cascade the subscription update
           if (newChildProps === lastChildProps.current) {
+            latestReduxStore.current = latestStoreState
```

Comment:
This line doesn't look right. We shouldn't be assigning refs while rendering - that's definitely _not_ safe for Concurrent Mode.

Reply:
Hi, that assignment is NOT on the render phase, is on the subscription Callback.
We have 2 case when store changes
1) the change produce a selector change -> we enqueque a component render with useState.setState(newReduxStore)
2) the change DOES not produce a selector change -> we "save" the new reduxState, because we have to always remember the last "reduxState event callback change" if there is an update from others reason like prop changes.

Reply:
My bad, sorry.

Reply:
this should be resolved, if you agree.